sample-demo-lambda-app/README.md

```diff
@@ -1,9 +1,73 @@
 # Amazon CodeGuru Profiler Python Demo Lambda Application
 
-Here's a sample code you can run in a Python Lambda in your account. Further instructions will follow soon.
-
-To enable Amazon CodeGuru Profiler, follow these instructions:
+To enable Amazon CodeGuru Profiler for a Lambda application, follow these instructions:
 * For Python 3.8: Enable `Code Profiling` for your Lambda's configuration in the `Monitoring and operation tools` tab: https://docs.aws.amazon.com/codeguru/latest/profiler-ug/setting-up-short.html#setting-up-step-2
 
 * For Python 3.6+: Follow these instructions for enabling Profiler https://docs.aws.amazon.com/codeguru/latest/profiler-ug/python-lambda.html
 
+Here's a sample code you can run in a Python Lambda in your account. You can follow the instructions to have a Python Lambda deployed in your account with CodeGuru Profiler enabled.
+
+### Create resources
+
+First step is to create the resources needed to run the application.
+
+* Go to the S3 console, create an S3 bucket and take a note of the name.
+    * Example of the name: profiler-python-recommendations-demo
```

Comment:
the S3 bucket name must be unique across all users and all regions so maybe add a comment about this or propose a different name like `profiler-python-recommendations-demo-your-name-here`
This is also why using S3 buckets in example code is not always easy. I tend to use DDB instead